test/unittests.jl

```diff
@@ -35,8 +35,8 @@ using Test
         @test ustrip(y) ≈ T(0.4)
 
         if R <: Rational
-            @test string(x) == "0.2 𝐋 ¹ 𝐌 ⁵ᐟ²"
-            @test string(inv(x)) == "5.0 𝐋 ⁻¹ 𝐌 ⁻⁵ᐟ²"
+            @test string(x) == "0.2 m kg⁵ᐟ²"
+            @test string(inv(x)) == "5.0 m⁻¹ kg⁻⁵ᐟ²"
```

Comment:
Not a physicist. And this is late in the piece for such a question, but in what situations are rational units used? What is `kg^{5/2}` ? So you can have (abusing notation) `y = sqrt(x)::m^{1/2}` and then `(y^2)::m`?

Reply:
I can't think of any constants or units that are rational either. (when represented in SI base units). Having the dimensions be rational is more for their use in intermediate calculations.

(You could use `FixedRational{Int8,1}` as the parametric type of `Dimensions` if you are confident the dimensions will never need to be rational)

Reply:
(Oops, I mean `Int8`. No need for `FixedRational` if you aren't using rational numbers...)